app/models/theme_settings_migration.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class ThemeSettingsMigration < ActiveRecord::Base
+  belongs_to :theme
+  belongs_to :theme_field
+
```

Comment:
We should proper add some AR validations to ensure fields cannot be null so that the error can be raised in the app before it hits the database.

Reply:
Done.